.github/workflows/release-mac.yml

```
@@ -210,17 +210,48 @@ jobs:
           
           echo "DMG_PATH=$DMG_PATH" >> $GITHUB_ENV
 
+      - name: Generate Appcast for Sparkle Updates
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
+          SPARKLE_PRIVATE_KEY: ${{ secrets.SPARKLE_PRIVATE_KEY }}
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:5:79: Double quote to prevent globbing and word splitting

(shellcheck)